jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/GettingStartedAuthUtils.kt

```diff
@@ -82,6 +83,10 @@ fun requestCredentialsForCodeWhisperer(
     isFirstInstance: Boolean = false,
     connectionInitiatedFromExplorer: Boolean = false
 ): Boolean {
+    if (isRunningOnRemoteBackend()) {
```

Comment:
ideally we disable the buttons too

Reply:
We won't be able to disable the link for logging in with Idc, which is why added the mesage here